picard/ui/filebrowser.py

```diff
@@ -168,6 +180,18 @@ def _get_destination_from_path(self, path):
             destination = os.path.dirname(destination)
         return destination
 
+    def load_selected_files(self):
+        indexes = self.selectedIndexes()
+        if not indexes:
+            return
+        paths = set(self.model.filePath(index) for index in indexes)
+        dirs = set(p for p in paths if os.path.isdir(p))
+        files = paths - dirs
+        tagger = QtCore.QObject.tagger
+        for d in dirs:
+            tagger.add_directory(d)
+        tagger.add_files(files)
```

Comment:
Not really related to this PR, but this code makes me things about it:

That's a long time I'm bothered by this separation between files and directories in the code and in the UI (file selection).

At this point, it would probably make sense to unify `add_directory()` (which calls `add_files()`) and `add_files()` in one function `add_paths()` taking a list of paths, testing them for validity and handle files and directories.

It would reduce code redundancy (here introduced, because we already test for dirs elsewhere in order to call either `add_directory()` or `add_files()`.

What do you think?

Reply:
Yes, agreed. I actually thought about this, too, while writing the code. I'll look at this in a separate PR. I'm not sure yet if it will get a combined function, because we benefit from handling it in a single call function, or if it will just sit above the existing add_files / add_directories functions.